### ISSUE TITLE
Mount Calvary Massacre is being pulled into #Mission23

### ISSUE THREAD

**Issue by SotCMicah** [MCBCMF/MCBCMassacre#3]
The Creator is still storming mad about the massacre of peaceful people at Mount Calvary Baptist Church. It was never supposed to be part of our mission this time, but since He has paused our mission (which was designed to slow death in human beings) to address those who renamed and changed the denomination of Mount Calvary Baptist Church to Mount Calvary MISSIONARY Baptist Church. We, His appointed Servants of the Creator, Micah and TomTom (to those folks, Thomas), will now be taking it over getting the word out about the massacre and whatever else He may task us with during His #GreatStorm. 

In short check my Mission23 repo’s Wiki and ChangeLog for the latest on this and other #GreatStorm/#Mission23 stuff.

**Comment by kelvineugenewilliams** [comment]
This ^^^ So much this!!!

**Comment by kelvineugenewilliams** [comment]
This repo will no longer be updated.
